cmd/clean.go

```diff
@@ -3,28 +3,47 @@ package cmd
 import (
 	"bytes"
 	"fmt"
+	"log"
+	"os"
+	"strings"
+
 	"github.com/kubefirst/kubefirst/configs"
+	"github.com/kubefirst/kubefirst/internal/aws"
 	"github.com/kubefirst/kubefirst/internal/reports"
+	"github.com/kubefirst/kubefirst/pkg"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	"log"
-	"os"
-	"strings"
 )
 
-// todo delete the s3 buckets associated with the ~/.kubefirst file
 // todo ask for user input to verify deletion?
 // todo ask for user input to verify?
 // cleanCmd removes all kubefirst resources locally for new execution.
 var cleanCmd = &cobra.Command{
 	Use:   "clean",
 	Short: "removes all kubefirst resources locally for new execution",
-	Long: `Kubefirst creates files and folders during installation at your local environment. This command removes and 
-re-create all Kubefirst files.`,
+	Long: `Kubefirst creates files, folders and cloud buckets during installation at your environment. This command removes and 
+re-create all Kubefirst files. To destroy cloud resources you need to specify aditional flags (--destroy-buckets)`,
 	Run: func(cmd *cobra.Command, args []string) {
 
 		config := configs.ReadConfig()
 
+		destroyBuckets, err := cmd.Flags().GetBool("destroy-buckets")
+		if err != nil {
+			log.Panic(err)
+		}
+		destroyConfirm, err := cmd.Flags().GetBool("destroy-confirm")
+		if err != nil {
+			log.Panic(err)
+		}
+		if destroyBuckets && !destroyConfirm {
+			destroyConfirm = pkg.AskForConfirmation("This process will delete cloud buckets and all files inside, do you really want to proceed?")
+			if !destroyConfirm {
+				os.Exit(130)
```

Comment:
out of curiosity, what does the 130 means?

Reply:
https://www.linuxdoc.org/LDP/abs/html/exitcodes.html

Reply:
130 | script terminated by Control-C
-- | --

Reply:
User desired to abort execution.